nixos/modules/services/games/openarena.nix

```diff
@@ -0,0 +1,61 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  name = "OpenArena";
+  cfg = config.services.openarena;
+in
+{
+  options = {
+    services.openarena = {
+      enable = mkEnableOption name;
+
+      openPorts = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to open firewall ports for " + name;
+      };
+
+      extraFlags = mkOption {
+        type = types.string;
```

Comment:
`types.string` is deprecated. Please use `types.listOf types.str` and `concatStringsSep " " cfg.extraFlags` below